packages/smart-wallet/src/smartWallet.js

```diff
@@ -41,14 +41,25 @@ const mapToRecord = map => Object.fromEntries(map.entries());
  * @see {@link ../README.md}}
  */
 
-// One method yet but structured to support more. For example,
-// maybe suggestIssuer for https://github.com/Agoric/agoric-sdk/issues/6132
-// setting petnames and adding brands for https://github.com/Agoric/agoric-sdk/issues/6126
 /**
  * @typedef {{
  *   method: 'executeOffer'
  *   offer: import('./offers.js').OfferSpec,
- * }} BridgeAction
+ * }} ExecuteOfferAction
+ */
+
+/**
+ * @typedef {{
+ *   method: 'tryExitOffer'
+ *   offerId: import('./offers.js').OfferId,
+ * }} TryExitOfferAction
+ */
+
+// One method yet but structured to support more. For example,
```

Comment:
there are now two methods. the part about additional potential ones is worth keeping.